game_loader.py

```diff
@@ -0,0 +1,91 @@
+from email.policy import default
+from importlib.resources import path
+from operator import le
+from operator import mul
+import os
+import textwrap
+import click
+from pyfiglet import Figlet
+from crpg import generate
+from functools import reduce
+from os.path import exists
+
+#Allows adding specifications when starting program; at the moment, only add the abitly to specify exact file path for .dl
+@click.command()
+@click.option("--path", default="game_libary/", help="Provide file path to a .dl file not located under /game_libary")
+def start(path):
+    print(path)
+    #Possible could be more elegant, checks if user specified path; if path exist runs it
+    if(path == "game_libary/"):
+        dl = selectDL(path)
```

Comment:
can you catch the "FileNotFoundError" and respond with a descriptive message before exiting? (using the function `exit(1)` perhaps)

Reply:
I set it up so that if it can't find the specified path, it asks the user to enter a new path. Is it still necessary to add a "FileNotFoundError"?

Reply:
Could you abstract the entire section to just deal with the path regardless of if it is "game_libary/" with an exception? You could encase it in a while loop so that it always asks only if the exception doesn't go off.

That would help simplify lines 19-42.